raiden_contracts/contract_manager.py

```diff
@@ -93,6 +95,8 @@ def _compile_all_contracts(self) -> None:
         old_working_dir = Path.cwd()
         chdir(_BASE)
 
+        subprocess.run(["make", "-C", "..", "solve_templates"])
```

Comment:
The pypi package will **not** & IMO should **not** contain the makefile, therefore this would actually break.

I think generating the contracts out of templates shouldn't be done in the contract manager at all... the contracts should be generated manually and then packaged into the pypi package (excluding the templates)

Reply:
Oh I see.  I didn't think about that.

Reply:
Now I've removed these `make` calls.

The sources will be added in the package in the next PR: https://github.com/raiden-network/raiden-contracts/pull/614/files#diff-97c91a104c431d0c365565d3ac03ac13R12 .  I'm not adding templates.